extensions/wikia/Chat2/js/emoticons.js

```diff
@@ -101,7 +101,7 @@ if(typeof EmoticonMapping === 'undefined'){
 					currentKey = url;
 					//$().log("  " + url + "...");
 				} else if (self._settings[currentKey]) {
-					var glyphMatch = emoticonArray[i].match(/^\*\*[ ]*([^*].*)/); // line starting with 2 "**"'s then optional spaces, then some non-empty content.
+					var glyphMatch = emoticonArray[i].match(/^\*\* *([^*"][^"]*)/); // line starting with 2 "**"'s then optional spaces, then some non-empty content.
```

Comment:
I'm not sure if preventing only again ```"``` is ideal solution.
I remember really old IE treating `` ` `` as a delimeter so old IE is still vulnerable here.
https://groups.google.com/d/msg/null-co-in/iqxoB7PXT5A/fKIjAsijn0kJ

Escaping output which @Grunny suggested sounds like more bulletproof solution.